doc/source/contributing.rst

```diff
@@ -0,0 +1,89 @@
+.. _ref_contribute:
+
+##########
+Contribute
+##########
+
+Overall guidance on contributing to a PyAnsys library appears in the
+`Contributing <https://dev.docs.pyansys.com/how-to/contributing.html>`_ topic
+in the *PyAnsys Developer's Guide*. Ensure that you are thoroughly familiar
```

Comment:
```suggestion
in the *PyAnsys developer's guide*. Ensure that you are thoroughly familiar
```
Recent decision to follow Google style guidelines for using sentence case titles.